docs/requirements.txt

```diff
@@ -11,6 +11,7 @@ jsonpickle
 sortedcontainers
 texar
 texar-pytorch
+spacy
```

Comment:
We usually don't put 3rd packages to the core requirements. Users only need to install them when needed

Reply:
docs fail if I don't do this. Any idea why that happens?